evalml/tests/automl_tests/test_automl_utils.py

```diff
@@ -115,6 +113,16 @@ def test_make_data_splitter_error():
     y = X.pop('target')
 
     with pytest.raises(ValueError, match="problem_configuration is required for time series problem types"):
-        make_data_splitter(X, y, ProblemTypes.TIME_SERIES_REGRESSION, n_splits=5, shuffle=False)
+        make_data_splitter(X, y, ProblemTypes.TIME_SERIES_REGRESSION)
     with pytest.raises(KeyError, match="Problem type 'XYZ' does not exist"):
-        make_data_splitter(X, y, 'XYZ', n_splits=5, shuffle=False)
+        make_data_splitter(X, y, 'XYZ')
+
+
+def test_make_data_splitter_error_shuffle_random_state():
+    n = 10
+    X = pd.DataFrame({'col_0': list(range(n)),
+                      'target': list(range(n))})
+    y = X.pop('target')
+
+    with pytest.raises(ValueError, match="Setting a random_state has no effect since shuffle is False."):
```

Comment:
Does our `TrainingValidationSplit` also raise this exception?

Reply:
Nope, I don't think so!